Project/Source/FileSystem/TextureImporter.h

```diff
@@ -12,6 +12,7 @@ class TextureImportOptions : public ImportOptions {
 
 public:
 	bool flip = false;
+	TextureCompression compression = TextureCompression::NONE;
 	TextureWrap wrap = TextureWrap::REPEAT;
 	TextureMinFilter minFilter = TextureMinFilter::LINEAR_MIPMAP_LINEAR;
 	TextureMagFilter magFilter = TextureMagFilter::LINEAR;
```

Comment:
I would put the values that we are going to normally use as the default.

Reply:
This is the default. You'll see that compressed textures have a lot of artifacts.